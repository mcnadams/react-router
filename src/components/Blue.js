import React from 'react';

export default function Blue() {
  const style = {
    background: 'blue',
    width: '250px',
    height: '250px'
  };

  return <div style={style}></div>;
}
