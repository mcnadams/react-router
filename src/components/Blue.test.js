import React from 'react';
import { shallow } from 'enzyme';
import Blue from './Blue';

describe('Blue', () => {
  it('Blue', () => {
    const wrapper = shallow(<Blue />);
    expect(wrapper).toMatchSnapshot();
  });
});
