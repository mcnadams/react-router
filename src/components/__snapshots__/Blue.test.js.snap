// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blue Blue 1`] = `
<div
  style={
    Object {
      "background": "blue",
      "height": "250px",
      "width": "250px",
    }
  }
/>
`;
